src/utils/common.js

```diff
@@ -46,12 +46,36 @@ import {
   BITCOIN_ADDRESS_PREFIX,
 } from 'constants/assetsConstants';
 import { MANAGE_USERS_FLOW } from 'constants/navigationConstants';
+import { isProdEnv, isTest } from './environment';
+
 
 const WWW_URL_PATTERN = /^www\./i;
 const supportedAddressPrefixes = new RegExp(
   `^(?:${ETHEREUM_ADDRESS_PREFIX}|${BITCOIN_ADDRESS_PREFIX}):`, 'gi',
 );
 
+export const printLog = (...params: any) => {
+  if ((isProdEnv && !__DEV__) || isTest) return;
+  console.log(...params);
+};
```

Comment:
Just curious of what would be the benefit of having this. Is that bad to call `console.log` on other environment than dev or test?

Reply:
Exactly, it is known that console.log produces huge lags in real devices.

https://reactnative.dev/docs/performance.html#using-consolelog-statements

Reply:
Wouldn't it be better to just remove them by using the suggested plugin?

Reply:
Well, tbh both results the same, but we can add it. However with this approach it will just remove one line that we have now.